### DISCUSSION TITLE
GLIBC

### DISCUSSION THREAD

**Discussion by Pacojim** [ory/hydra#2428] [Q&A] [answered]
I have a problem with instalation Hydra: "version `GLIBC_2.28' not found (required by ./hydra)".
How can I solve this problem? Upgrading the GLIBC is usually a very bad idea and I don't know what to do.
Thanks.

**Comment by aeneasr** [comment]
Rum the non sqlite version and it should work!

**Comment by zepatrik** [comment, marked as answer]
I guess you should update your distro then, glibc 2.28 was released in Aug 2018. In general, glibc has few breaking changes anyways, and quite some security patches with every release.
You can also try `patchelf`:

```shell
$ ./patchelf --set-interpreter /path/to/newglibc/ld-linux.so.2 --set-rpath /path/to/newglibc/ hydra
```

(taken from https://stackoverflow.com/a/44710599)

**Comment by vinckr** [comment]
Hey @Pacojim,
Just checking in if your question was answered or if there was still something unclear.
I will mark the discussion as answered for now.

Thanks a ton! 🐝
